circe-akka-serializer/src/main/scala/org/virtuslab/ash/circe/CirceTraitCodec.scala

```diff
@@ -85,8 +76,7 @@ trait CirceTraitCodec[Ser <: AnyRef] extends Codec[Ser] {
     .toMap
     .withDefaultValue("")
 
-  def manifest(o: AnyRef): String = parentsUpToRegisteredTypeMap(o.getClass.getName)
-
+  // Decoder apply method - decodes from Json into an object of type Ser
```

Comment:
Nit: I'd even change it into a proper `/** ... */` scaladoc, same below

Reply:
done for both